python/mxnet/image/image.py

```diff
@@ -85,6 +84,46 @@ def imread(filename, *args, **kwargs):
     return _internal._cvimread(filename, *args, **kwargs)
 
 
+def imresize(src, w, h, interp):
+    r"""Resize image with OpenCV.
+
+    Note: `imread` uses OpenCV (not the CV2 Python library).
```

Comment:
can you change this `imresize`, I assuming you mean `imresize` internally uses `imread` which uses OpenCV
saying `imread` is a little confusing when you read the doc of this API alone

Reply:
Note has a special syntax to appear on the website, i think its `::Note::` can you please use that?